core/tests/test_utils.py

```diff
@@ -622,6 +622,9 @@ def process_and_flush_pending_tasks(self, queue_name=None):
                 if task.url == '/_ah/queue/deferred':
                     from google.appengine.ext import deferred
                     deferred.run(task.payload)
+                elif feconf.EMAILS_TASK_PREFIX in task.url:
+                    # Flush task if task is of emails taskqueue.
```

Comment:
Yeah I understand the problem we will have in future with these settings. Actually I myself was finding a way to convert dict to string but I never gave it much attention.
Let  me see what I can do to convert params in payload.
Thanks for the tip.